### DISCUSSION TITLE
Feature request: MIDI "strum" capability

### DISCUSSION THREAD

**Discussion by evilknish** [SynthstromAudible/DelugeFirmware#2292] [Ideas]
Hi all, 

Supporter and big believer in the community devs! What's been accomplished so far has been incredible.

Wondering if someone can add a "strum" feature, like the one found on the Oxi One in this vid https://www.youtube.com/watch?v=oBb0mXGT5s4 at 6:58?

Since there's no chord pack mode on the Deluge, I was thinking it could be implemented by having the user input there own chord (two or more notes placed at the exact same time). Then holding down the note from that chord you want to hit first for the strum, and using one of the knobs or buttons to toggle strum for that chord. Any way this is possible? Would anyone like this feature? It would save time from having to manually slightly nudge each note of a chord into a strum, for sure. 

Thanks!

**Comment by Dust-r** [comment]
Great idea!

**Reply by duncanrmi** [reply]
this seems like it would be very useful- could it be done by simply spreading the note-start timing of a chord, starting with the lowest or highest note & incrementally delaying them?
at the moment, I do this all manually & it's pretty tedious..... 
I think I'd want it to scatter the note timings with a degree of randomness too.
